en/log/flight_review.md

```diff
@@ -0,0 +1,332 @@
+# Flight Review 101
+
+The [Flight Review](http://logs.px4.io) plots can be used to analyze vehicle condition in a specific flight.
+
+The plots are meant to be self-explanatory, but it takes some experience to know what ranges are acceptable and what a plot should look like.
+This page explains how to interpret the plots and identify common problems.
+
+## General Usage
+
+Features that are common to many plots:
+
+- Plot background color is used to indicate flight mode during recording (where graphs depend on mode):
+  ![Flight Modes](../../assets/flight_log_analysis/flight_review/flight_modes.png)
+  - **Flight mode:** Background colour on the body of the plot indicates flight mode.
+    Hovering with the mouse over a plot shows the flight mode labels.
+  - **VTOL flight mode:** VTOL vehicles additionally show the VTOL mode as background colour in the bottom part of the plot (blue for multicopter, yellow for fixed wing, and red for transition).
+- Mouse scrolling on a particular plot axis zooms that axis (horizontally or vertically).
+- Mouse scrolling inside the plot zooms both axes.
+
+## PID Tracking Performance {#tracking}
+
+Depending on the flight mode, the vehicle controllers may attempt to track position, velocity, altitude or rate setpoints (the tracked setpoints depend on the mode, e.g.: in Stabilized mode there is no velocity setpoint).
+
+The **Estimated** line (red) should closely match with the **Setpoint** (green).
+If they do not, in most cases the PID gains of that controller need to be tuned.
+
```

Comment:
It would be good to have one or two graph images here. Maybe a "good track" and an unacceptable overshoot. Just to give a flavour. You could reuse image from the tuning guide.

Reply:
Fair enough. Personally I'm OK with 1 or 2 images here - purpose isn't to explain so much as position people in the right charts "mentally".

I'll set resolved and you decide if you want to add or not.